src/azure-cli/azure/cli/command_modules/containerapp/custom.py

```diff
@@ -1223,8 +1223,17 @@ def update_containerappsjob_logic(cmd,
                     scale_rule_def["auth"] = auth_def
                     if not scale_def:
                         scale_def = JobScaleModel
```

Comment:
Wait, `scale_def` is miss, please check logic in line 1204-1209

![image](https://github.com/Azure/azure-cli/assets/46143499/54f9ee5a-2c3e-4146-895b-abe2fc0ed597)

Reply:
@Greedygre I have updated the code to avoid assigning `scale_def` again with an empty model. That will not be required.